### DISCUSSION TITLE
Unable to delete S3 Object

### DISCUSSION THREAD

**Discussion by hanielu** [pocketbase/pocketbase#2946] [Q&A]
I've set up an s3 bucket and a corresponding policy with the following permissions: PutObject, DeleteObject, and GetObject.

After filling in the necessary credentials, I get the following error, and I have no idea why. I've look in the S3 dashboard and I see the test object has been added, but for some reason, pocketbase can't delete them.

Any Help or nudge in the right direction will be greatly appreciated

![image](https://github.com/pocketbase/pocketbase/assets/28874974/2ed7bfba-6d1b-47b0-8e49-d811406c0a16)

**Comment by MuslemRahimi** [comment]
Do you have an IAM user with S3FullAccess. This looks like one of those errors when you don't have Access
